services/app-api/utils/constants/constants.ts

```diff
@@ -56,66 +56,103 @@ export const bucketTopics = {
+    "state_ownershipControlDisclosureWebsiteLink",
+    "program_contractTitle",
+    "program_type",
+    "program_type-otherText",
+    "program_coveredSpecialBenefits",
+    "program_coveredSpecialBenefits-otherText",
+    "program_specialBenefitsAvailabilityVariation",
+    "program_encounterDataUses",
+    "program_encounterDataUses-otherText",
+    "program_encounterDataSubmissionCorrectionPerformanceEvaluationCriteria",
+    "program_encounterDataSubmissionCorrectionPerformanceEvaluationCriteria-otherText",
+    "program_encounterDataSubmissionCorrectionPerformanceEvaluationCriteriaContractLanguageLocation",
+    "program_encounterDataSubmissionQualityFinancialPenaltiesContractLanguageLocation",
+    "program_encounterDataQualityIncentives",
+    "program_criticalIncidentDefinition",
+    "program_standardAppealTimelyResolutionDefinition",
+    "program_expeditedAppealTimelyResolutionDefinition",
+    "program_grievanceTimelyResolutionDefinition",
+    "state_bssWebsite",
+    "state_bssEntityServiceAccessibility",
+    "state_bssEntityLtssProgramDataIssueAssistance",
+    "state_bssEntityPerformanceEvaluationMethods",
+    "plan_medicalLossRatioPercentageAggregationLevel",
+    "plan_medicalLossRatioPercentageAggregationLevel-otherText",
+    "plan_populationSpecificMedicalLossRatioDescription",
+    "plan_medicalLossRatioReportingPeriodStartDate",
+    "plan_medicalLossRatioReportingPeriodEndDate",
+    "program_encounterDataSubmissionTimelinessStandardDefinition",
   ],
-  wildcard: ["appeals", "grievances", "sanctions"],
+  accessMeasures: [
+    "id",
+    "name", // TODO do access measures have names? Delete this line if not.
+    "accessMeasure_generalCategory",
+    "accessMeasure_standardDescription",
+    "accessMeasure_standardType",
+    "accessMeasure_standardType-otherText",
+    "accessMeasure_providerType",
+    "accessMeasure_providerType-otherText",
+    "accessMeasure_applicableRegion",
+    "accessMeasure_applicableRegion-otherText",
+    "accessMeasure_population",
+    "accessMeasure_population-otherText",
+    "accessMeasure_monitoringMethods",
+    "accessMeasure_monitoringMethods-otherText",
+    "accessMeasure_oversightMethodFrequency",
+    "accessMeasure_oversightMethodFrequency-otherText",
+  ],
+  plans: [
+    "id",
+    "name",
+    "plan_medicalLossRatioPercentageAggregationLevel",
+    "plan_medicalLossRatioPercentageAggregationLevel-otherText",
+    "plan_populationSpecificMedicalLossRatioDescription",
+    "plan_medicalLossRatioReportingPeriodStartDate",
+    "plan_medicalLossRatioReportingPeriodEndDate",
+    "program_encounterDataSubmissionTimelinessStandardDefinition",
+    "plan_programIntegrityReferralPath",
+    "plan_smaMfcuConcurrentProgramIntegrityReferrals",
+    "plan_smaMfcuAggregateProgramIntegrityReferrals",
+    "plan_beneficiaryCircumstanceChangeReportingFrequency",
+  ],
+  qualityMeasures: [
+    "id",
+    "name", // TODO do quality measures have names? Delete this line if not.
```

Comment:
Assuming this todo is for the future, not for today?

Reply:
It's today if you (or @cassandradanger ) know the answer to the question. I don't and I'm kind of swamped. There's no harm if `"name"` stays in this list despite not existing on entities - it's just less tidy.

Reply:
i don't believe accessMeasures, qualityMeasures,  have a "name" key, just "id" and "label"

Reply:
I did not see a `name` key in them either when I attempted to fill out everything I could